style.css

```diff
@@ -309,6 +283,58 @@ header {
   background: #008552;
 }
 
+/* PopUp modal style */
+.modal-back {
+  background: #c1c7d060;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  z-index: 100;
+  top: 0;
+  padding: 4%;
+  overflow: auto;
+  backdrop-filter: blur(5px);
+}
+
+.modal {
+  background: #fff;
+  padding: 16px;
+  position: relative;
+  border-radius: 8px;
+  margin: 0 auto;
+  width: 100%;
+}
+
+.modal figure {
+  border-radius: 8px;
+  overflow: hidden;
+  margin-bottom: 16px;
+  height: auto;
+  min-height: 220px;
+  background: #eaf2fd;
+}
+
+.modal div ul.language {
+  margin-bottom: 12px;
+  justify-content: flex-start;
+}
+
+.modal div p {
+  margin-bottom: 16px;
+}
+
+.close-popup {
```

Comment:
- [ ] The close button on the mobile view is not displaying correctly, please consider fixing this so the button is shown on the mobile 

- Your design:
![Screenshot_20220513_063605](https://user-images.githubusercontent.com/25227022/168217908-787b9c1f-df9f-43e2-b461-91f65a21ccfe.png)

- Figma design:
![Screenshot_20220513_063538](https://user-images.githubusercontent.com/25227022/168217816-cd6360ac-7318-45b0-9b95-56299ad35920.png)